purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -788,9 +791,41 @@ class Purchases internal constructor(
     }
     // endregion
 
+    /**
+     * Redeem a web purchase using a [DeepLink.WebPurchaseRedemption] object obtained
+     * through [Purchases.parseAsDeepLink].
+     */
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    fun redeemWebPurchase(webPurchaseRedemption: DeepLink.WebPurchaseRedemption, listener: RedeemWebPurchaseListener) {
+        purchasesOrchestrator.redeemWebPurchase(webPurchaseRedemption, listener)
+    }
+
+    /**
+     * Represents a valid RevenueCat deep link.
+     */
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    sealed class DeepLink {
```

Comment:
Could be a sealed interface, but nabd. (Same in `customEntitlementComputation`.)